src/ansys/fluent/core/solver/flobject.py

```diff
@@ -727,6 +740,27 @@ def __call__(self, **kwds):
         newkwds = _get_new_keywords(self, kwds)
         return self.flproxy.execute_cmd(self._parent.path, self.obj_name, **newkwds)
 
+    def _get_value_quantity_map(self, **kwds):
+        newkwds = _get_new_keywords(self, kwds)
+        results_list = self.flproxy.execute_cmd(self._parent.path, self.obj_name, **newkwds)
+        unit_quantity_map = {}
+        for quantity in range(len(results_list)):
+            for unit_quantity, real in results_list[quantity].items():
+                quantity = unit_quantity
+                quantity = quantity.replace("_", ' ')
+                quantity = ' '.join(word.capitalize() for word in quantity.split())
+                value = real[0]
+                unit_quantity_map[quantity] = 1.0
+        return [value, unit_quantity_map]
+
+    def value(self, **kwds):
+        value_map = Command._get_value_quantity_map(self, **kwds)
+        return value_map[0]
+
+    def quantity_map(self, **kwds):
```

Comment:
It is probably more useful to have this method on the returned object of command/query execution rather than on the command/query object itself.

Reply:
Yes, we will discuss this soon.

Reply:
@hpohekar it would be good to resolve this issue.

Reply:
@seanpearsonuk @mkundu1 We already tried this for `report_definitions`. The `report_definitions` returns `list` type of object, then we will have to modify default built-in python data type to add `quantity_map` as an attribute to `list` type.

![image](https://user-images.githubusercontent.com/106588300/213136592-b015391d-d65a-40ef-a149-263ef46f2840.png)


![image](https://user-images.githubusercontent.com/106588300/213136765-35bc99dc-48b4-4afd-a929-37e5875c71b8.png)

Similar to this we will have to implement for other data types.

Reply:
@seanpearsonuk @mkundu1  but as per our latest discussion we do not want to depend on return data type of commands and query.

Reply:
Done. Thank you.